o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=98) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=98) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=99) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=99) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=100) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=100) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=101) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=101) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=102) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=102) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=103) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=103) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=104) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=104) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=105) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=105) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=106) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=106) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=107) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=107) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
o.m.jdbc.message.server.ErrorPacket -- Error: 1045-28000: Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
com.mayer.playground.Playground -- PRoblem, Exception Message: (conn=108) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
java.sql.SQLInvalidAuthorizationSpecException: (conn=108) Access denied for user 'Maximilian'@'172.17.0.1' (using password: NO)
	at org.mariadb.jdbc.export.ExceptionFactory.createException(ExceptionFactory.java:286)
	at org.mariadb.jdbc.export.ExceptionFactory.create(ExceptionFactory.java:370)
	at org.mariadb.jdbc.client.impl.ConnectionHelper.authenticationHandler(ConnectionHelper.java:287)
	at org.mariadb.jdbc.client.impl.StandardClient.<init>(StandardClient.java:185)
	at org.mariadb.jdbc.Driver.connect(Driver.java:70)
	at org.mariadb.jdbc.Driver.connect(Driver.java:101)
	at org.mariadb.jdbc.Driver.connect(Driver.java:27)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at com.mayer.playground.Playground.lambda$main$0(Playground.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
